test/libsolidity/syntaxTests/string/string_multipart_hex_valid_parts.sol

```diff
@@ -0,0 +1,8 @@
+contract test {
+    function f() public pure returns (bytes32) {
+        bytes32 escapeCharacters = hex"aa" hex"ax";
```

Comment:
Perhaps move the invalid `hex"ax"` to another invalid hex string test and replace this by a "valid", but too short hex string? It took me a second to realize that the `x` in the second part makes this a hex string with uneven nibbles.  But I'm also fine with the current test.